ch_tools/chadmin/cli/table_group.py

```diff
@@ -363,3 +369,59 @@ def materialize_ttl_command(ctx, dry_run, all_, database, table, exclude_table):
         materialize_ttl(
             ctx, database=t["database"], table=t["table"], echo=True, dry_run=dry_run
         )
+
+
+@table_group.command("set-flag")
+@option(
+    "--database",
+    type=StringParamType(),
+    help="Filter tables to set the flag by the specified database name.",
+)
+@option(
+    "--table",
+    type=StringParamType(),
+    help="Filter in tables by the specified table name.",
+)
+@option(
+    "--exclude-table",
+    type=StringParamType(),
+    help="Filter out tables by the specified table name.",
+)
+@option("--flag", type=str, default="convert_to_replicated", help="Flag name.")
+@option(
+    "-v",
+    "--verbose",
+    type=bool,
+    is_flag=True,
+    help="Show tables and flag paths.",
+)
+@pass_context
+def set_flag_command(
+    ctx: Context,
+    database: str,
+    table: str,
+    exclude_table: str,
+    flag: str,
+    verbose: bool,
+) -> None:
+    """
+    Create a flag with the specified name inside the data directory of the table.
+    """
+    if database is None:
+        return
+
+    tables = (
+        get_tables_to_convert(ctx, database, table, exclude_table)
+        if flag == CONVERT_TO_REPLICATED_FLAG
+        else get_tables_dict(ctx, database, table, exclude_table)
+    )
```

Comment:
We can also make this command more generic and make user's responsibility to filter tables with not MergeTree engine, but currently there are no other table flags in clickhouse other than convert_to_replicated.

Reply:
I like this idea. Let's do it.

And in this case I'd suggest to replace `--flag` `@option` with mandatory `@argument`.

Then setting the flag `convert_to_replicated` for all tables will look something like this: 
```
chadmin table set-flag --all --exclude-database system,_system --engine %MergeTree --exclude-engine Replicated% convert_to_replicated
```